test/1.4/errors/T_WPRAGMA.dml

```diff
@@ -0,0 +1,13 @@
+/*
+  © 2024 Intel Corporation
+  SPDX-License-Identifier: MPL-2.0
+*/
+dml 1.4;
+
+device test;
+
+// no warning
+/*% COVERITY irrelevant %*/
+
+/// WARNING WPRAGMA
+/*% MYVERYCOOLPRAGMA %*/
```

Comment:
Some other corners:
```
/*% 😀 strange character in pragma tag %*/
/*%     COVERITY weird spacing%*/
/*% COVERITY forgot closing % */
//% COVERITY this isn't a pragma, right?
```
I don't have a strong opinion on which ones should be considered valid vs invalid vs discouraged, but they deserve tests since they now are language corners.

Reply:
Done.